cernopendata/modules/fixtures/data/records/delphi-documentation-manuals.json

```diff
@@ -0,0 +1,205 @@
+[
+  {
+    "abstract": {
+      "description": "Detailed description of the DELPHI conditions database contents, and the tools coming with it."
+    },
+    "accelerator": "CERN-LEP",
+      "collaboration": {
+      "name": "DELPHI collaboration",
+      "recid": "27050"
+    },
+    "date_created": [
+      "1993"
+    ],
+    "date_published": "2024",
+    "distribution": {
+      "formats": [
+        "pdf"
+      ],
+      "number_files": 3
+    },
+    "experiment": [
+      "DELPHI"
+    ],
+    "files": [
+      {
+        "checksum": "adler32:18d0bebc",
+        "size": 404470,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/documentation/ddb/cargo600.pdf"
+      },
+      {
+        "checksum": "adler32:9ef43714",
+        "size": 229767,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/documentation/ddb/ddapp600.pdf"
+      },
+      {
+        "checksum": "adler32:63ea0076",
+        "size": 557435,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/documentation/ddb/dplot.pdf"
+      }
+    ],
+    "license": {
+      "attribution": "CC0"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "27001",
```

Comment:
We can use values around 80500 so that we don't fill "old holes" with the new records.  I.e. from 80501 to 80504.

Reply:
Fixed